src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/PrimitiveGhostData.hpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/VariablesTag.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+template <typename T>
+class Variables;
+/// \endcond
+
+namespace grmhd::ValenciaDivClean::subcell {
+/*!
+ * \brief Computes the rest mass density \f$\rho\f$, pressure \f$p\f$,
+ * \f$u_i\f$, magnetic field \f$B^i\f$, and the divergence cleaning field
+ * \f$\Phi\f$ on the subcells so they can be used for reconstruction.
+ *
+ * The computation copies the data from the primitive variables tag to a
+ * new Variables and computes \f$W v^i\f$. In the future we will likely want
```

Comment:
Can you clarify how "W v^i" is related to the "u_i" you mention above? At first glance these things seem inconsistent... is the "u_i" above wrong? 😛

Reply:
The `brief` should read `W v^i` (which is `\gamma^{ij} u_j`). I'll update the brief :)